alchimia/__init__.py

```diff
@@ -1,10 +1,12 @@
 from __future__ import absolute_import, division
 
-from alchimia.strategy import TwistedEngineStrategy, TWISTED_STRATEGY
+from alchimia.strategy import TWISTED_STRATEGY
+from alchimia.engine import TwistedEngine
 
+wrap_engine = TwistedEngine.from_sqlalchemy_engine
```

Comment:
`help(alchimia.wrap_engine)` shows the function's name as `from_sqlalchemy_engine`.  Maybe it's worth naming the class method that?

Reply:
Honestly I think I'm OK with this; `help` is not really the recommended documentation system anyway :).  perhaps we'll use a fancier way of aliasing at some point later.